master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -1325,6 +1330,19 @@ private[celeborn] class Master(
     workerExcludeResponse.getSuccess -> sb.toString()
   }
 
+  def removeWorkersUnavailableInfo(unavailableWorkers: Seq[WorkerInfo]): HandleResponse = {
```

Comment:
We need also use this way in `timeoutWorkerUnavailableInfos`?

Reply:
thanks, addressed.